make-scripts/start_bridge.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if ! command -v npm &> /dev/null
+then
+    echo "npm could not be found, install npm first"
+    exit
+fi
+
+cd ./bridge/
+
+# install the dependencies
+npm install
+
+# accept the API URL and API TOKEN as the user inputs
+read -p 'Enter API URL: ' API_URL
+read -sp 'Enter API Token: ' API_TOKEN
```

Comment:
FYI, the API_TOKEN can be left out and it will be automatically pulled via `kubectl` if not specified.

Reply:
So whether to pull it or not depends on whether the `API_TOKEN`  env var exists or not, right?

Reply:
The check is formulated in JavaScript:

https://github.com/keptn/keptn/blob/7d5fb6e6174a13c7fdfe45650266d18a1fab6780/bridge/server/app.js#L9-L23

I'm pretty sure javascript will evaluate an empty string as false, so it doesn't matter if the variable exists or not. So in the best case you do not need to handle this case in the code, just add a note here :)

Reply:
sure!